lore/encoders.py

```diff
@@ -36,8 +35,9 @@ def __init__(self, column, name=None, dtype=numpy.uint32, embed_scale=1, tags=[]
         self.column = column
         self.dtype = dtype
         self.embed_scale = embed_scale
+
         self.tags = tags
-        
+        self.twin = twin
```

Comment:
We need to provide this by default in `__setstate__` if not present for backward compatibility. e.g. from estimators.py:
```
    def __setstate__(self, dict):
        self.__dict__ = dict
        backward_compatible_defaults = {
            'towers': 1,
            'cudnn': False,
            'multi_gpu_model': None,
            'output_activation': 'sigmoid',
        }
        for key, default in backward_compatible_defaults.items():
            if key not in self.__dict__.keys():
                self.__dict__[key] = default
```